src/components/shift-schedule/ScheduleCell.tsx

```diff
@@ -0,0 +1,58 @@
+import { ScheduleCellDialog } from '@/components/shift-schedule/ScheduleCellDialog';
+import { Dialog, DialogContent, DialogTrigger } from '@/components/ui/Dialog';
+import { TableCell } from '@/components/ui/Table';
+import { cx } from '@/lib/utils';
+import { UserIcon, UsersIcon } from 'lucide-react';
+import { useTranslations } from 'next-intl';
+
+type ScheduleCellProps = {
+  tDialog: {
```

Comment:
Dont need this as props, you can get the translations inside the component isntead so we avoid prop drilling